internal/sirius/get_deputy_clients.go

```diff
@@ -204,7 +205,7 @@ func restructureOrders(apiOrders apiOrders) Orders {
 		reformattedDate := formatDate(t.OrderDate)
 
 		var supervisionLevel string
-		if t.LatestSupervisionLevel != nil {
+		if t.LatestSupervisionLevel.SupervisionLevel.Label != "" {
```

Comment:
Is it possible for `LatestSupervisionLevel` to be null? I imagine this would error when trying to get `SupervisionLevel` if the `LatestSupervisionLevel` came back as null.

Reply:
I've added a test to cover this, thanks @hiboabd